### PR TITLE
chore: changed the visual look, added emojis, and edited grammar.

### PR DESCRIPTION
**Pull request by NoorinSakhi**
Please take a look at my changes and see if you approve thank you.

### CODE REVIEW

File: english/covid-19/0/0-C.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
-You cried for help 📢 as loud as you can. But, seems like no one is around 🤷‍♂️. 
+## You cried for help 📢 as loud as you can. 
+### But, seems like no one is around 🤷‍♂️. 
 
-You thought about your family 👪 and how they must be worried about you 😟. What will you do?
+*You thought about your family 👪 and how they must be worried about you 😟. What will you do?*
 
-[🚪 Find a way out](../WIP.md)
+- [ ] [🚪 Find a way out](../WIP.md)
```
Comment (ming-tsai) [inline english/covid-19/0/0-C.md:6]: Please remove `[ ]`
```suggestion
- [🚪 Find a way out](../WIP.md)
```

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @NoorinSakhi,
Everything is good, but please remove all `[ ]` to all files

**Comment by NoorinSakhi** [comment]
I removed all the ```[]```. Thank you.

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @NoorinSakhi,
Sorry I didn't see you are converting the text to tile, and it doesn't look very well, so my recommendation `add the emojis and edited the grammar`

**Review by ming-tsai — APPROVED** [review]
Hi @NoorinSakhi,
Nice work

**Pull request merged**